lib/sort-lines.js

```diff
@@ -52,6 +52,18 @@ function lineLengthReverseCompare(a, b) {
   return a.length > b.length ? -1 : 1;
 }
 
+function getVariableCharacters(line) {
+  return (line.match(/(.*)=/) || []).pop();
```

Comment:
Won't this be weird if the variables are a mix of const/var?

```js
const a = 1;
const b = 11;
var c = 2;
var d = 22;
```

Reply:
Why would anyone mix `const` and `var`? Maybe `let` and `const`, but arguably separating them makes the code more readable. For example, I think this is awful:
```
let x = 1;
const b = 2;
const def = 4;
let y = 423523;
```

vs 
```
let x = 1;
let y = 423523;
const b = 2;
const def = 4;
```